detection_rules/main.py

```diff
@@ -149,12 +151,48 @@ def view_rule(ctx, rule_id, rule_file, api_format):
     if not rule:
         client_error('Unknown format!')
 
-    click.echo(toml_write(rule.rule_format()) if not api_format else
-               json.dumps(rule.contents, indent=2, sort_keys=True))
+    if verbose:
+        click.echo(toml_write(rule.rule_format()) if not api_format else
+                   json.dumps(rule.contents, indent=2, sort_keys=True))
 
     return rule
 
 
+@root.command('export-rules')
+@click.argument('rule-id', nargs=-1, required=False)
```

Comment:
for consistency, should `rule-id` be an `option` instead of an argument?
don't forget to update the docs too if you do

Reply:
I actually made it an argument to be consistent with the `view-rule` and `validate-rule` commands, where id is an optional argument and file is an option